ui/app/i18n/clinical/locale_en.json

```diff
@@ -343,5 +343,8 @@
     "PROGRAM_DOCTORINCHARGE": "Doctor In-Charge",
     "PROGRAM_PATIENT_STAGE": "Patient Stage",
     "CLINICAL_CLASS": "Class",
-    "CLINICAL_CASTE": "Caste"
+    "CLINICAL_CASTE": "Caste",
+    "NO_PACS_STUDY_FOR_ORDER": "Can not identify study for order. Please verify PACS configuration",
+    "RADIOLOGY_STUDY_FETCH_ERROR": "Can not fetch radiology studies. Please verify PACS configuration",
```

Comment:
"Can not" => "Unable to fetch.. "  or "Unable to identify".  might be better choice.

Reply:
done.